packages/excalidraw/components/App.tsx

```diff
@@ -2568,6 +2572,11 @@ class App extends React.Component<AppProps, AppState> {
         EVENT.MOUSE_MOVE,
         this.updateCurrentCursorPosition,
       ),
+      addEventListener(
+        document,
+        EVENT.POINTER_MOVE,
+        this.updateCurrentCursorPosition,
+      ),
```

Comment:
I don't recall why we're using `MOUSE_MOVE` above. I feel something will break, but anyway, let's remove the `MOUSE_MOVE` and keep `POINTER_MOVE` only?

What were you fixing with `POINTER_MOVE` in the first place? I assume it didn't work on mobile?